aws_lambda_powertools/utilities/parser/models/apigwv2.py

```diff
@@ -55,17 +55,25 @@ class RequestContextV2(BaseModel):
     timeEpoch: datetime
     http: RequestContextV2Http
 
+    # Validator to normalize the timeEpoch field
+    # Converts the provided timestamp value to a UTC datetime object
+    # See: https://github.com/pydantic/pydantic/issues/6518
+    @validator("timeEpoch", pre=True)
+    def coerce_timestamp(cls, value):
+        date_utc = datetime.fromtimestamp(int(value) / 1000, tz=timezone.utc)
+        return date_utc
```

Comment:
The Pydantic project will revert the behavior back to what we had in v1, so we can remove this validator after that.

Issue: https://github.com/pydantic/pydantic/issues/6518

Reply:
They fixed this bug in v2.0.3 - https://github.com/pydantic/pydantic/releases/tag/v2.0.3.

We have removed this extra code now.